### ISSUE TITLE
bug(database): confd log error

### ISSUE THREAD

**Issue by aledbf** [deis/deis#3436]
Using `deis/database:v1.5.0` I see this error:

```
Apr 08 22:28:51 nodo-3 sh[13633]: 2015-04-08T22:28:51Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928281/298901368]) [298929280]
Apr 08 22:28:53 nodo-3 sh[13633]: 2015-04-08T22:28:53Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928362/298904039]) [298929361]
Apr 08 22:28:53 nodo-3 sh[13633]: 2015-04-08T22:28:53Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928362/298901368]) [298929361]
Apr 08 22:28:55 nodo-3 sh[13633]: 2015-04-08T22:28:55Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928451/298904039]) [298929450]
Apr 08 22:28:55 nodo-3 sh[13633]: 2015-04-08T22:28:55Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928451/298901368]) [298929450]
Apr 08 22:28:57 nodo-3 sh[13633]: 2015-04-08T22:28:57Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928547/298904039]) [298929546]
Apr 08 22:28:57 nodo-3 sh[13633]: 2015-04-08T22:28:57Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928547/298901368]) [298929546]
Apr 08 22:28:59 nodo-3 sh[13633]: 2015-04-08T22:28:59Z 55b2af686f67 confd[152]: ERROR 401: The event in requested index is outdated and cleared (the requested history has been cleared [298928631/298904039]) [298929630]
```

From the `etcd` (docs)[https://github.com/coreos/etcd/blob/master/Documentation/api.md] this could be a `confd` error using watch?

**Comment by aledbf** [comment]
ping @bacongobbler

**Comment by bacongobbler** [comment]
@alebdf this looks like https://github.com/kelseyhightower/confd/issues/205 which is resolved in https://github.com/kelseyhightower/confd/pull/254.

**Comment by mboersma** [comment]
This should be fixed in the next release of `confd` v0.9.0.
